src/workloads/docs/Loader.yml

```diff
@@ -0,0 +1,83 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+# More threads requires a larger connection pool.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 200
+
```

Comment:
Could you add a comment up here explaining that there are two ways to use this actor. The first instantiation shows the default, and the second shows MultipleThreadsPerCollection: true. Use the second if you have more loader threads than collections to load.

Reply:
Sure thing, more documentation is good.

Reply:
done